invenio_config_tugraz/config.py

```diff
@@ -325,3 +325,34 @@
 
 See https://docs.sqlalchemy.org/en/latest/core/engines.html.
 """
+
+
+RDM_PERMISSION_POLICY = TUGrazPermissionPolicy
+
+TUGRAZ_CURATORS = {
```

Comment:
is it possible to move that to the invenio-rdm-pure module? for me the invenio-config-tugraz should serve as the base for the configuration and therefore other modules like invenio-rdm-pure should depend on invenio-config-tugraz and not vice versa

Reply:
the policy can be overwritten from every module as well as the curators - this should not be a problem